storage/innobase/xtrabackup/test/suites/binlog/binlog_10.sh

```diff
@@ -0,0 +1,6 @@
+. inc/common.sh
+. inc/binlog_common.sh
+vlog "------- TEST 10 -------"
+INDEX_FILE="$topdir/binlog-dir1/bin.index"
+FILES="$topdir/binlog-dir1/bin.000003"
```

Comment:
how do we know that  binlog with .00003 will be created in this test?

Reply:
Hi @satya-bodapati I've not modified the original binlog.sh test. I just split it into single files.
FILES and INDEX_FILE is tested on restore, to validate if the file exists. By looking into all tests, I can see that all that specify a file with the numeric part uses .000003 or .000004 - Most likely those are the starting point on MTR (1 at --initialize, 1 at startup, 1 as part of flush engine logs 1 as part of startup after the restore) = 4

Reply:
ok, test knows upfront that it will rotate the binlogs :+1: